faker/providers/phone_number/ar_SA/__init__.py

```diff
@@ -0,0 +1,113 @@
+# coding=utf-8
+from __future__ import unicode_literals
+import re
+
+from .. import Provider as PhoneNumberProvider
+
+
+class Provider(PhoneNumberProvider):
```

Comment:
can you add unit tests for the new provider?

Reply:
Will do.